OpenLIFUTransducerTracker/Resources/UI/OpenLIFUTransducerTracker.ui

```diff
@@ -107,8 +107,8 @@
          </item>
          <item>
           <widget class="QLabel" name="photoscanGenerationStatusMessage">
-           <property name="text">
-            <string>(placeholder_photoscangeneration_status_label)</string>
+           <property name="styleSheet">
+            <string notr="true">font-style: italic</string>
```

Comment:
Post-review change: I made this italic to match the other one. I feel like progress status looks better in italic since it makes it feel like a dynamic text somehow